src/styles/reset.css

```diff
@@ -0,0 +1,39 @@
+
+/*
```

Comment:
Why does this stay in css?

Reply:
It's a reset css, they are short and straightforward. It applies some defaults to avoid weird CSS behavior that may occur later. It doesn't need the extra benefits "scss" provides